src/apis/storage_partitioning.js

```diff
@@ -95,6 +95,19 @@ export const partitioningSetEncrypt = ({ encrypt, partitioning }) => {
             });
 };
 
+/**
+ * @param {string} partitioning     DBus path to a partitioning
+ */
+export const partitioningSetHomeReuse = ({ partitioning }) => {
+    return getPartitioningRequest({ partitioning })
```

Comment:
I think this is nicer writen with async await. however I know we still use a lot of then so not objection if you want to keep it like this.

For the sake of it, that would be 

``` js
export const partitioningSetHomeReuse = async ({ partitioning }) => {
  const request = await getPartitioningRequest({ partitioning });
  ....
  await setPartitioningRequest({ partitioning, request });;
};
```

Reply:
Updated